client/my-sites/earn/ads/wrapper.jsx

```diff
@@ -181,6 +162,15 @@ class AdsWrapper extends Component {
 		);
 	}
 
+	renderEmptyContent() {
+		return (
+			<EmptyContent
+				illustration="/calypso/images/illustrations/illustration-404.svg"
+				title={ this.props.translate( 'You are not authorized to view this page.' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **28** times:
`translate( 'You are not authorized to view this page' )` *ES Score: 8*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

Reply:
Switching to remove full stop (accidentally put comma in the commit message, oops) since that's the one that's used elsewhere. :)